docs/MAINTAINERS.adoc

```diff
@@ -0,0 +1,200 @@
+Their main responsibility is to triage and review the incoming pull requests,
+and to guide newcomer contributors who are not familiar with the project's processes.
+GitHub team: link:https://github.com/orgs/jenkinsci/teams/core-pr-reviewers[@jenkinsci/core-pr-reviewers].
+
+**Core Maintainers** get `Write` permissions in the repository, and hence they are able to merge pull requests.
+Their responsibility is to perform pull request reviews on a regular basis and to merge ready pull requests towards the weekly releases (`master` branch).
+They are also responsible to monitor the weekly release status and to perform triage of critical issues.
+GitHub team: link:https://github.com/orgs/jenkinsci/teams/core[@jenkinsci/core].
+
+**Release Team Members** are responsible for Jenkins weekly and LTS releases.
+Led by the link:https://jenkins.io/project/team-leads/#release[Jenkins Release Officer], they initiate releases, prepare changelogs and backport changes into the link:TODO[Stable branch].
+Team members get `Admin` permissions in the Jenkins core repository, and they also get permissions to trigger release Pipelines.
+
+=== Ladder
+
+* *Contributors*. Anyone can participate.
+  There is no precondition except having a GitHub account, just submit pull requests or comment in existing ones!
+* **Core Pull Request Reviewers** can be nominated by contributors in the link:https://groups.google.com/d/forum/jenkinsci-dev[developer mailing list]. 
+Self-nomination is fine.
+Decision is made by a consensus in the mailing list or via voting at the link:https://jenkins.io/project/governance-meeting/[governance meeting].
+* **Core Maintainers** - same process as for Core PR reviewers.
+  All nominees must also sign an link:https://github.com/jenkinsci/infra-cla/[Individual Contributor License Agreement] before getting the permission in GitHub repositories.
+* **Release Team Members** are assigned by the Jenkins Release officer
+
+== Pull request review process
+
+Jenkins core is a mission-critical part of the ecosystem.
+We need to ensure that submitted pull requests are not only code complete,
+but also that they do not introduce undesired defects, breaking changes and technical debt.
+At the same time, we are interested to make the review process as simple as possible for contributors and maintainers.
+
+=== Code review goals
+
+There are the following code review goals:
+
+* Ensure that the pull request is feasible and that it fits the Jenkins ecosystem
+* Verify compatibility of the suggested changes
+* Ensure that the pull request matches the code quality practices: test automation, Javadoc for newly introduced APIs, etc.
+* Ensure that the pull request contains appropriate changelogs and, if needed, upgrade guidelines.
```

Comment:
This doesn't have its own section below, as the only item in this list. Deliberate?

Reply:
yes. Can be expanded later if needed